components/tabs/tabs.js

```diff
@@ -828,12 +914,15 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 	_updateMeasures() {
 		let totalTabsWidth = 0;
 		if (!this.shadowRoot) return;
-		const tabs = [...this.shadowRoot.querySelectorAll('d2l-tab-internal')];
+		const tabs = this._defaultSlotBehavior ? [...this.shadowRoot.querySelectorAll('d2l-tab-internal')] : this._tabs;
 
 		const tabRects = tabs.map((tab) => {
+			const tabRect = tab.getBoundingClientRect();
+			const offsetLeft = this._defaultSlotBehavior ? tab.offsetLeft : this.#getOffsetLeft(tab, tabRect);
```

Comment:
This `offsetLeft` issue was interesting. Since the `offsetParent` of the `tab` is no longer the container but actually the document body, these calculations were no longer as expected. I made a calculation that seems to be returning the same result (seen in `#getOffsetLeft`) so let me know thoughts on that.

Another option is to set `position: relative` on `d2l-tabs` but I'm unsure if that would have other repercussions. This also would need additional work in the `rtl` case to account for `ext` slot content, and would need additional modification if there were any other slot additions etc., so that seems fragile.

Reply:
I'm guessing it is the `body` in the demo page, but it could be some other element in the consumers context. 

The container is still `position: relative` ya, but there's some complication in that it's in a different DOM scope? I feel like we've run into `offsetParent` issues before. Might be worth checking out our [getOffsetParent](https://github.com/BrightspaceUI/core/blob/main/helpers/dom.js#L164). Maybe it helps?